src/domUpdates.js

```diff
@@ -258,12 +258,14 @@ const createIngredientsHTML = ingredients => {
 const searchForRecipes = () => {
   const data = {
     ourRecipes: recipeData,
-    yourRecipes: null
+    yourRecipes: currentUser.recipesToCook
   }
   let searchedRecipes = searchRecipes(data[pageData.currentView], ingredientsData, searchBar.value)
```

Comment:
data[pageData.currentView] 

WOW

Reply:
Just BTW, this will get changed to recipesOfInterest when we update our data model